tools/PowerShell/Microsoft.WinGet.Client/src/Library/Enable-WinGetLocalManifest.ps1

```diff
@@ -0,0 +1,41 @@
+Function Enable-WinGetLocalManifest
+{
+    <#
+        .SYNOPSIS
+        Enables the Windows Package Manager to work with local manifest files. 
+        
+        .DESCRIPTION
+        By running this cmdlet the Windows Package Manager will configured to support working with manifests on the file system.
+
+        .PARAMETER VerboseLog
+        Used to provide verbose logging for the Windows Package Manager.
+    #>
+
+    PARAM {
+        [Parameter()] [switch]                  $VerboseLog
+    }
```

Comment:
Should this be just `-Verbose`?

Reply:
I don't think that works
```
PS C:\WINDOWS\system32> winget settings --enable LocalManifestFiles -Verbose
Windows Package Manager v1.1.12986
Copyright (c) Microsoft Corporation. All rights reserved.

Argument alias was not recognized for the current command : '-Verbose'

Open settings in the default json text editor. If no editor is configured, opens settings in notepad. For available settings see https://aka.ms/winget-settings This command can also be used to set administrator settings by providing 'enable' or 'disable' argument

usage: winget settings [<options>]

The following options are available:
  --enable   Enables the specific administrator setting
  --disable  Disables the specific administrator setting

More help can be found at: https://aka.ms/winget-settings
```

Reply:
I think that `-Verbose` is an inherent property if it has a cmdlet binding, and since the settings command for winget doesn't support a verbose option, I don't think it needs to be included here at all

Reply:
`-Verbose` is a standard parameter, so if the intent of this is just to create verbose user output, then it need not be a separate parameter here.

At least with normal functions, you enable this by making the function an advanced function. Just add `[CMDLETBINDING]` just above the parm block.

I initially thought this switch might be a special diag log for winget? Useful to clarify that.

Reply:
I think it is intended to be a diagnostic log for winget, but I don't think the setting command supports the diagnostic logging in winget. It's also probably not important to have a cmdlet binding here, since there is nothing using `Write-verbose`. Since these functions are just a wrapper for winget commands anyways, I would argue that keeping it simple at the start is better. If the verbose functionality is wanted or needed, then it can be added later